index.js

```diff
@@ -1,60 +1,29 @@
-const {app, BrowserWindow, ipcMain, Menu, shell} = require('electron')
 const defaultMenu = require('electron-default-menu')
+const window = require('electron-window')
+const electron = require('electron')
+const Env = require('envobj')
```

Comment:
it seems pretty lightweight and sets it up so its easier in the future to add environment variables. I think it's fine to keep it if we think we're going to have more env vars.